<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
    background-color: rgb(20, 0,20);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGraphicsView" name="boardGraphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="piecesGraphicsView">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>BLOKUS REMASTERED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="piecesLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Pieces</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>531</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="playerStatshorizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="p1verticalLayout">
       <item>
        <widget class="QLabel" name="p1label">
         <property name="text">
          <string>Player 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="p1line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p1nameLabel">
         <property name="text">
          <string>p1_name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="placedLabel">
         <property name="text">
          <string>Placed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remainingLabel">
         <property name="text">
          <string>Remaining:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="p2verticalLayout">
       <item>
        <widget class="QLabel" name="p2label">
         <property name="text">
          <string>Player 2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="p2line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2nameLabel">
         <property name="text">
          <string>p2_name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="placedLabel_2">
         <property name="text">
          <string>Placed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remainingLabel_2">
         <property name="text">
          <string>Remaining:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="p3verticalLayout">
       <item>
        <widget class="QLabel" name="p3label">
         <property name="text">
          <string>Player 3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="p3line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3nameLabel">
         <property name="text">
          <string>p3_name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="placedLabel_3">
         <property name="text">
          <string>Placed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remainingLabel_3">
         <property name="text">
          <string>Remaining:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="p4verticalLayout">
       <item>
        <widget class="QLabel" name="p4label">
         <property name="text">
          <string>Player 4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="p4line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p4nameLabel">
         <property name="text">
          <string>p4_name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="placedLabel_4">
         <property name="text">
          <string>Placed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remainingLabel_4">
         <property name="text">
          <string>Remaining:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>191</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="AbilityverticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Abilities</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bombPushButton">
       <property name="text">
        <string>Bomb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirrorPushButton">
       <property name="text">
        <string>Copy Cat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="WildPushButton">
       <property name="text">
        <string>Design Your Own</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="instructLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>510</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>How To Play:</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructLabel_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>530</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rules: After the first block of yours is played, each of your next placements MUST connect via a diagonal to one of your already placed pieces.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="instructLabel_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Game ends when no player can make a move and has expended all Placing Abilities (Mirror and DYO).</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="instructLabel_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>640</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Press 'r' to rotate, or use your scroll wheel.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="instructLabel_5">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>660</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Right click or ESC to cancel.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>630</y>
      <width>191</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="SkipPushButton">
       <property name="text">
        <string>Skip Turn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EndGamePushButton">
       <property name="text">
        <string>End Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
